packages/main/src/components/FilterGroupItem/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { usePassThroughHtmlProps } from '@ui5/webcomponents-react-base/lib/usePassThroughHtmlProps';
+import { createComponentStyles } from '@ui5/webcomponents-react-base/lib/createComponentStyles';
+import { BusyIndicator } from '@ui5/webcomponents-react/lib/BusyIndicator';
+import { BusyIndicatorSize } from '@ui5/webcomponents-react/lib/BusyIndicatorSize';
+import { FlexBox } from '@ui5/webcomponents-react/lib/FlexBox';
+import { Label } from '@ui5/webcomponents-react/lib/Label';
+import React, { forwardRef, ReactElement, RefObject } from 'react';
+import { CommonProps } from '../../interfaces/CommonProps';
+import styles from './FilterGroupItem.jss';
+
+const useStyles = createComponentStyles(styles, { name: 'FilterGroupItem' });
+
+export interface FilterGroupItemPropTypes extends CommonProps {
+  children: ReactElement;
+  label?: string;
+  groupName?: string;
+  labelTooltip?: string;
+  loading?: boolean;
+  mandatory?: boolean;
```

Comment:
naming: This prop is called `required` in the label component. Should we adapt the same name here?